examples/Node/file.js

```diff
@@ -21,8 +21,8 @@ detector.on('silence', function () {
   console.log('silence');
 });
 
-detector.on('sound', function () {
-  console.log('sound');
+detector.on('sound', function (buffer) {
+  console.log('sound', buffer);
```

Comment:
Perhaps we don't have to send the buffer data to console? I'm just afraid that people may get scared by the binary data from console :-) Perhaps, adding a comment of how people can possibly use this buffer data will be enough?